components/post.js

```diff
@@ -106,7 +106,7 @@ const Post = ({
                 src={img.thumbnails?.large?.url || img.url}
                 loading="lazy"
                 width={img.thumbnails?.large?.width}
-                height={img.thumbnails?.large?.height}
+                height={img.thumbnails?.large?.height > 384 ? 600 : img.thumbnails?.large?.height}
```

Comment:
This isn’t correct—you should pass the dimensions of the source image, then use CSS to control the size on the page itself. That way, before the image downloads, the browser can calculate how big the box for it should be & do all the layout around it, without having to recalculate the whole layout after each image comes in

Reply:
the problem is with the max height of the image being set in the css, there's just this massive padding at the bottom...

hmm let me try doing it in css then